source/settings/config-file/config-file-options.txt

```diff
@@ -50,13 +50,41 @@ You can configure the following settings in a configuration file:
 
 .. list-table::
    :header-rows: 1
-   :widths: 30 70
+   :widths: 40 60
 
    * - Setting 
      - Definition
 
+   * - autoUpdates
+     - Allow |compass-short| to periodically check for new updates.
+
+   * - :ref:`enableDevtools <compass-enable-dev-tools>`
+     - Enable Chrome DevTools in |compass-short|.
+
+   * - enableMaps
+     - Allow |compass-short| to make requests to third-party mapping services.
+
+   * - :ref:`enableShell <disable-shell>`
+     - Enable or disable the embedded MongoDB Shell on |compass-short|.
+
    * - :ref:`forceConnectionOptions <compass-force-connection-options>`
      - .. include:: /includes/fact-force-connection-options.rst
+    
+   * - ignoreAdditionalCommandLineFlags 
+     - Allow or disallow additional command-line option flags.
+
+   * - installURLHandlers 
+     - Register |compass-short| as a handler for mongodb:// and mongodb+srv:// 
+       URLs. 
+
+       If :guilabel:`Install Compass as URL Protocol Handler` is enabled, 
+       you can open |compass-short| by navigating to a mongodb:// or 
+       mongodb+srv:// URL in your browser.
```

Comment:
Just checking, we mean an internet browser here, correct?

Reply:
Yep 👍 Will update with that clarification.